### PR TITLE
Issue/11714

### PR DESCRIPTION
**Pull request by cehsu**
These changes incorporate the latest round of stakeholder feedback as documented [here](#11714). These changes also set up the [schema](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11499).

### CODE REVIEW

File: src/schemas/complaint-tool/schema.js [hunk 1]
```diff
@@ -83,38 +82,35 @@ let schema = {
...
-        postalCode: {  // TYPE: text (5)
-          type: 'string',
-          pattern: '^\\d{5}$' // common definition pattern (meets submission requirements)
-        },
-        country: { // TODO: determine validation, picklist? 
-          type: 'string',
-          'enum': ['US'] // Only US addresses are supported
-        }
-      }
+    // TRANSLATE: address and address2 must be combined into a single comma separated string property named 'address'
```
Comment (erikphansen) [inline src/schemas/complaint-tool/schema.js]: would it be useful to add an example of what the final combined `address` should look like given sample values for the input `address` and `address2` fields? Such as "`given 'address': '123 Main St.' and 'address2': 'APT 23' then the final translated 'address' should be '123 Main St., APT 23'`

File: dist/complaint-tool-schema.json [hunk 2]
```diff
@@ -39,97 +39,141 @@
...
+      ]
+    },
+    "postalCode": {
+      "type": "string",
+      "pattern": "^\\d{5}$"
+    },
+    "country": {
+      "type": "string",
+      "enum": [
+        "US"
```
Comment (rianfowler) [inline dist/complaint-tool-schema.json]: Some [schools](url
https://www.vets.gov/gi-bill-comparison-tool/profile/31100792) are not in the US

Comment (cehsu) [reply dist/complaint-tool-schema.json]: Thanks @rianfowler, stakeholders indicated that supporting international addresses should be revisited in the future. We can add a comment about that here, and double check with them if of interest.
Another thought relating to this is whether we should display the country field at all if there is only one choice allowed, and simply set this on the BE. But perhaps that is a question for design, and I will ask on the contact page PR.

File: src/schemas/complaint-tool/schema.js (hunk 1 above)
Comment (cehsu) [reply src/schemas/complaint-tool/schema.js]: @lihanli are we planning to perform these translations on the BE? If so, would anything in addition to these "TRANSLATE" comments be helpful?

Comment (lihanli) [reply src/schemas/complaint-tool/schema.js]: an example should be fine

Comment (cehsu) [reply src/schemas/complaint-tool/schema.js]: Great idea @erikphansen, updated.

**Pull request merged**